fleetshard/pkg/central/charts/data/rhacs-operator/templates/rhacs-operator-deployment.yaml

```
@@ -78,13 +82,11 @@ spec:
                   containerName: manager
                   resource: limits.memory
                   divisor: '0'
-            - name: OPERATOR_CONDITION_NAME
-              value: rhacs-operator.v3.74.0
-            {{ if .Values.operator.centralLabelSelector -}}
+            {{- if $.Values.operator.centralLabelSelector -}}
             - name: CENTRAL_LABEL_SELECTOR
-              value: {{ .Values.operator.centralLabelSelector | quote }}
+              value: {{ $.Values.operator.centralLabelSelector }}
```

I must be missing something... we're deploying multiple controllers but they all share the same central label selector? :thinking:
Also, I don't see it being set anywhere. Is this dead code?

---

This probably needs to be something like
```suggestion
              value: "rhacs.redhat.com/version={{ .tag | lower }}"
```

---

or 
```suggestion
              value: {{ substr 0 64 (lower (printf "rhacs.redhat.com/version=" $.Values.operator.centralLabelSelector)) }}
```

---

@porridge I added this bit (CENTRAL_LABEL_SELECTOR) a few weeks ago, as I assumed that there would be 1 helm release per operator. But as @SimonBaeumer explained, it was actually decided to use 1 helm release for all operators.